tests/performance/tensor_eval/tensor_instructions.rb

```diff
@@ -0,0 +1,87 @@
+# Copyright Verizon Media. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+require 'performance_test'
+require 'app_generator/search_app'
+require 'environment'
+
+class TensorInstructionBenchmarkTest < PerformanceTest
+
+  COST = 'cost'
+  IMPL_LONG = 'implementation'
+  IMPL_SHORT = 'impl_short'
+  BM_TYPE = 'benchmark_type'
+
+  def setup
+    super
+    set_owner("arnej")
+  end
+
+  def test_instruction_benchmark
+    set_description("Test performance of low-level tensor instructions")
+    deploy_app(SearchApp.new.sd(selfdir+"foobar.sd"))
+    searchnode = vespa.search['search'].first
+    output = searchnode.execute("#{Environment.instance.vespa_home}/bin/vespa-tensor-instructions-benchmark")
```

Comment:
I think we need an option to the program where we run with fewer implementations and fewer benchmark cases. Currently we have ~80 benchmark cases and 6 implementations. Could also consider to graph only a subset.

Reply:
yes, that's WIP in another PR...

Reply:
👍